File: 프로그래머스/lv0/120875. 평행/평행.py
def solution(dots):
    answer = 0
    ans1 = (dots[0][0]-dots[1][0])/(dots[0][1]-dots[1][1])
    ans2 = (dots[2][0]-dots[3][0])/(dots[2][1]-dots[3][1])
    ans3 = (dots[0][0]-dots[2][0])/(dots[0][1]-dots[2][1])
    ans4 = (dots[1][0]-dots[3][0])/(dots[1][1]-dots[3][1])
    
    if ans1 == ans2 or ans3 == ans4 :
        return 1
    return 0

File: 프로그래머스/0/120875. 평행/평행.js
const solution = dots => {
    const [[x1, y1], [x2, y2], [x3, y3], [x4, y4]] = dots
    if ((x2 - x1) / (y2 - y1)  === (x4 - x3) / (y4 - y3)) return 1
    if ((x3 - x1) / (y3 - y1) === (x4 - x2) / (y4 - y2)) return 1
    
    
    
    return 0
}